ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/port-channel-short-esi.yml

```
@@ -16,6 +6,8 @@ servers:
         switch_ports: [ Ethernet10, Ethernet10]
         switches: [ port-channel-short-esi, Leaf1]
```

The test output does not contain these port-channels because `switches` need to contain `host1` + one other dummy name (keep Leaf1). Same below
```suggestion
        switches: [ host1, Leaf1]
```

---

Fixed